Traceback (most recent call last):
  File "main.py", line 5, in <module>
    n = int(input())
EOFError: EOF when reading a line
